roles/development/tasks/python.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: install packages for peresonal use via pip
   pip:
-    name: 'flake8,pep8,"ipython<5",pylint'
```

Comment:
ipython 5 and above doesn't work with ipdb 0.10.2. Also newer ipdb doesn't work for me on CentOS, so I had this workaround

Reply:
pip was throwing errors at me saying it was an invalid requirement, I'll see if I can get to the bottom of that.